Fix title of article to include 'and above' :cloud_with_lightning_and_rain: (#957)
MicrosoftDocs/iis-docs: iis/manage/configuring-security/how-to-set-up-ssl-on-iis.md

---
title: "How to Set Up SSL on IIS 7 and above"
author: rick-anderson
description: "The steps for configuring Secure Sockets Layer (SSL) for a site are the same in IIS 7 and above and IIS 6.0, and include the following: Get an appropriate ce..."
ms.date: 11/22/2007
ms.assetid: dbf635a5-921f-4488-99a2-bd1c2d1c0402
msc.legacyurl: /learn/manage/configuring-security/how-to-set-up-ssl-on-iis
msc.type: authoredcontent
---

> # How to Set Up SSL on IIS 7 and above

Reviewer: ```suggestion
# How to Set Up SSL on IIS 7 or later
```
